acceptance/suite_test.go

```diff
@@ -0,0 +1,274 @@
+//go:build acceptance
+// +build acceptance
+
+package influxtest
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ahmetalpbalkan/dlog"
+	"github.com/colega/envconfig"
+	influxdb "github.com/influxdata/influxdb-client-go/v2"
+	influxdb_api "github.com/influxdata/influxdb-client-go/v2/api"
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+	promapi "github.com/prometheus/client_golang/api"
+	promv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/stretchr/testify/suite"
+)
+
+var (
+	suiteContainerLabels = map[string]string{"acceptance-suite": "influx-proxy"}
+)
+
+// These tests verify that the Influx proxy is able to take in InfluxDB line protocol,
+// correctly parse it and convert it into a timeseries, and write the timeseries to
+// Cortex. Several services are run to execute the tests. The InfluxDB client is used to
+// send line protocol to the proxy. The proxy service accepts the line protocol, parses it,
+// and writes it to the Cortex service. The Prometheus client and API are used to query
+// Prometheus to verify that the line protocol was parsed and converted into the expected
+// timeseries, and that the timeseries was successfully written.
+
+func TestSuite(t *testing.T) {
```

Comment:
small tweak so that the documentation generator will pick up this comment.  (we don't generate docs automatically, but we could, and this would be a nice comment to see if we do)

```suggestion
// TestSuite verifies that the Influx proxy is able to take in InfluxDB line protocol,
// correctly parses it and converts it into a timeseries, and writes the timeseries to
// Cortex. Several services are run to execute the tests. The InfluxDB client is used to
// send line protocol to the proxy. The proxy service accepts the line protocol, parses it,
// and writes it to the Cortex service. The Prometheus client and API are used to query
// Prometheus to verify that the line protocol was parsed and converted into the expected
// timeseries, and that the timeseries was successfully written.
func TestSuite(t *testing.T) {
```